passive/ShipMuonShield.h

```diff
@@ -34,7 +34,7 @@ class ShipMuonShield : public FairModule
 
   void SetSupports(Bool_t supports) { 
     fSupport = supports;
-    FairLogger::GetLogger()->Warning(MESSAGE_ORIGIN, "Setting supports to %s. This will not have any effect if called after the geometry has been constructed.", fSupport ? "true" : "false");
+    LOG(INFO) <<"Setting supports to %s. This will not have any effect if called after the geometry has been constructed.", fSupport ? "true" : "false";

```

Comment:
Shouldn't this be `LOG(WARN)`? We should not change the severity of these, unless the change is intentional.

Reply:
yes, you are right. Will correct it.